modules/systemtests/src-test/org/ejbca/core/ejb/ca/caadmin/RenewCATest.java

```diff
@@ -97,6 +99,84 @@ public void test01renewCA() throws Exception {
         assertFalse(Arrays.equals(orgkey, newkey));
         log.trace("<test01renewCA()");
     }
+
+    
+    /** Test renewal of a CA using a different key algorithm. 
+     *  Note: Can run these tests alone by using: ant test:runone -Dtest.runone=RenewCATest
+    **/
+    
+    // Need these extra EJBs
+    private final CryptoTokenManagementSessionRemote cryptoTokenManagementSession = EjbRemoteHelper.INSTANCE
```

Comment:
Put these on the top of the file together with the other EJBs

Reply:
Fixed.